### ISSUE TITLE
add warm welcome bot to Slack community

### ISSUE THREAD

**Issue by fharper** [meilisearch/devrel#29]
A bot I created so I can welcome all new people joining our Slack, but only when I'm there and I have the time to discuss with them if they reply. The reply and Slack participations are way higher when I use it.

https://github.com/fharper/warm-welcome

**Comment by fharper** [comment]
Need to update my repo with a screenshot and all. I will propose to @CaroFG if she wants to be the one to welcome people or myself. Caro, I can quickly explain tomorrow during our standup

**Comment by ferdi05** [comment]
unsure that we really want to use a bot (for instance Orbit is doing a great job at this without using a bot), but there should be something to welcome Slack newcomers

**Comment by ferdi05** [comment]
Should explain what to talk about on some interesting channels like #product (not joined by default)

**Comment by CaroFG** [comment]
There is no product channel on Slack community 🤔

**Comment by ferdi05** [comment]
Indeed, but it's something we're thinking of (early discussion with @qdequele )
